docs/design-proposals/multi-tenancy/multi-tenancy-access-control.md

```diff
@@ -18,27 +18,54 @@ The key changes to each authenticator is that it needs to retrieve the tenant in
 
 The existing X509 certificate authenticator reads "/CN" and "/O" from cert subject, and map them to user name and groups respectively. 
 
-To make the authenticator tenancy-aware, "/O" will be mapped to tenant while "/OU" (Organization Unit) will be mapped to groups. 
+Below is a sample certificate request with the new format:
 
-Below is a sample cert:
+```text
+openssl req -new -key demo.pem -out demo-csr.pem -subj "/CN=user/O=app1/O=app2"
+```
+
+To make the authenticator tenancy-aware, "/O" will be changed to map to tenant while "/OU" (Organization Unit) to map to groups. To maintain backward compatibility, two formats are to be used. If "tenant:" is at the beginning of "/O", then what's after the it will be treated as tenant. For example, with "/O=tenant:tenantA/OU=app1/OU=app2", tenant will be set as "tenantA", and groups will have ["app1", "app2"]. If "tenant:" is not found at the beginning of "/O", the string before the first ":" in /CN will be used as tenant. For example, with "/CN=tenantB:demo/O=app1/O=app2", tenant will be set to "tenantB".
```

Comment:
Suggest listing the two examples in separate command lines just as it's done above. It's probably clearer?

Reply:
agree. fixing.